creator-node/src/routes/nodeSync.js

```diff
@@ -24,56 +24,92 @@ module.exports = function (app) {
    * }
    */
   app.get('/export', handleResponse(async (req, res) => {
+    // TODO - allow for offsets in the /export
+    const limit = 25000
     const walletPublicKeys = req.query.wallet_public_key // array
     const dbOnlySync = (req.query.db_only_sync === true || req.query.db_only_sync === 'true')
 
     const transaction = await models.sequelize.transaction()
     try {
       // Fetch cnodeUser for each walletPublicKey.
-      const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction })
+      const cnodeUsers = await models.CNodeUser.findAll({ where: { walletPublicKey: walletPublicKeys }, transaction, raw: true })
```

Comment:
oh beautiful, how much smaller does this make payload?

Reply:
also i'm not sure of diff btwn `raw = true` and `type = RAW`

Reply:
doesn't really change the payload size. what it does save us is the need to iterate through all the records and call .toJSON() on all of them. basically we remove four O(N) from unnecessary processing.

Reply:
reverted